public/angular/index.html

```diff
@@ -16,6 +16,24 @@
     <link rel="stylesheet" href="/css/loading-bar.min.css"/>
     <link rel="stylesheet" href="/css/tagify.css"/>
     <link rel="stylesheet" href="/css/spectrum.min.css"/>
+
+    <!-- Matomo -->
+    <script type="text/javascript">
+      var _paq = window._paq = window._paq || [];
+      /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+      _paq.push(["setCookieDomain", "*.aggiebeta.cartercenter.org"]);
```

Comment:
What are cookies used for here? We should also put this in `secrets.json` and document its purpose (many server admins wouldn't know what value to use here unless clearly explained).

Reply:
From my limited understanding, the cookies are useful for generating heatmaps and recorded sessions. I added another commit so that this property is also stored in secrets.json.example